app/hub/Views/SetupViews/SetupAntiSuiteView/SetupAntiSuiteViewContainer.jsx

```diff
@@ -94,44 +97,49 @@ class SetupAntiSuiteViewContainer extends Component {
 			enable_smart_block,
 			enable_ghostery_rewards,
 		} = this.props.setup;
+
+		const anti_tracking_enabled = IS_CLIQZ ? false : enable_anti_tracking;
+		const ad_block_enabled = IS_CLIQZ ? false : enable_ad_block;
+		const ghostery_rewards_enabled = !IS_CLIQZ;
 		const features = [
 			{
 				id: 'anti-tracking',
 				name: t('hub_setup_antisuite_name_antitracking'),
-				enabled: enable_anti_tracking,
-				toggle: () => {
-					this._handleToggle('anti-tracking');
-				},
-				description: t('hub_setup_antisuite_description_antitracking'),
+				enabled: anti_tracking_enabled,
+				locked: IS_CLIQZ,
+				toggle: IS_CLIQZ ?
+					() => {} :
+					() => this._handleToggle('anti-tracking'),
+				description: IS_CLIQZ ? t('hub_setup_feature_already_active') : t('hub_setup_antisuite_description_antitracking')
 			},
 			{
 				id: 'ad-block',
 				name: t('hub_setup_adblock_name_adblocking'),
-				enabled: enable_ad_block,
-				toggle: () => {
-					this._handleToggle('ad-block');
-				},
-				description: t('hub_setup_adblock_description_adblocking'),
+				enabled: ad_block_enabled,
+				locked: IS_CLIQZ,
+				toggle: IS_CLIQZ ?
+					() => {} :
+					() => this._handleToggle('ad-block'),
+				description: IS_CLIQZ ? t('hub_setup_feature_already_active') : t('hub_setup_adblock_description_adblocking'),
 			},
 			{
 				id: 'smart-blocking',
 				name: t('hub_setup_smartblocking_name_smartblocking'),
 				enabled: enable_smart_block,
-				toggle: () => {
-					this._handleToggle('smart-blocking');
-				},
+				toggle: () => this._handleToggle('smart-blocking'),
 				description: t('hub_setup_smartblocking_description_smartblocking'),
-			},
-			{
+			}
+		];
+
+		if (ghostery_rewards_enabled) {
```

Comment:
It looks like we're only using `ghostery_rewards_enabled` right here, would it be clearer to replace this with `!IS_CLIQZ` instead?

Reply:
Definitely agree. I've removed the variable entirely and replaced it with !IS_CLIQZ.